### PR TITLE
murmur-stats: Enhancements

### PR DESCRIPTION
**Pull request by brknkfr**
- python3 compatibility
- multigraph
- added muted/registered/unregistered users

### CODE REVIEW

File: plugins/mumble/murmur-stats [hunk 1]
```diff
@@ -14,31 +14,85 @@ iceslice='/usr/share/slice/Murmur.ice'
...
+  ice.destroy()
+  sys.exit(1)
+
+users = server.getUsers()
+users_muted = 0
+users_unregistered = 0
+users_registered = 0
+
+for key in list(users.keys()):
+  if users[key].userid == -1:
```
Comment (sumpfralle) [inline plugins/mumble/murmur-stats:72]: Maybe the next lines could be easier to understand with a list comprehension?
```python
users_unregistered = len(user for user in users.values() if user.userid == -1)
users_registered = len(user for user in users.values() if user.userid > 0)
users_muted = len(user for user in users.values() if user.mute or user.selfMute or self.suppress)
```

File: plugins/mumble/murmur-stats [hunk 2]
```diff
@@ -14,31 +14,85 @@ iceslice='/usr/share/slice/Murmur.ice'
...
+    users_unregistered += 1
+
+  if users[key].userid > 0:
```
Comment (sumpfralle) [inline plugins/mumble/murmur-stats:75]: A user ID with the value zero is not allowed?

File: plugins/mumble/murmur-stats [hunk 3]
```diff
@@ -14,31 +14,85 @@ iceslice='/usr/share/slice/Murmur.ice'
...
+
+  if users[key].selfMute:
+    users_muted += 1
+
+  if users[key].suppress:
+    users_muted += 1
+
+print("multigraph murmur_stats_users")
+print("users_total.value %i" % (len(users)))
+print("users_muted.value %i" % (users_muted))
```
Comment (sumpfralle) [inline plugins/mumble/murmur-stats:89]: Just for the sake of compeleteness: you are adding braces around `users_muted`, thus it *looks* like a tuple (as it is recommeneded for arguments of `%`), but it is still just a number :)

I would recommend one of the following approaches:
* A) remove the braces for clarity
* B) add a trailing comma within the braces in order to turn it into a tuple expression
* C) use one of the other (less suprising) string formatting methods (`"".format()` or `f""`)

**Review by sumpfralle — COMMENTED** [review]
Thank you for taking care for the python3-migration of this plugin!

Please take a look at my comments below.

File: plugins/mumble/murmur-stats (hunk 1 above)
Comment (brknkfr) [reply plugins/mumble/murmur-stats:72]: I'll have to add a list() for this.

File: plugins/mumble/murmur-stats (hunk 2 above)
Comment (brknkfr) [reply plugins/mumble/murmur-stats:75]: Oh, good catch. SuperUser normally has userid 0.

**Comment by sumpfralle** [comment]
Thank you for improving this plugin!

**Pull request merged**